x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorFiltering.java

```diff
@@ -121,6 +131,38 @@ public static ConnectorFiltering fromXContentBytes(BytesReference source, XConte
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private static final ConstructingObjectParser<List<ConnectorFiltering>, Void> CONNECTOR_FILTERING_PARSER =
+        new ConstructingObjectParser<>(
+            "connector_filtering_parser",
+            true,
+            args -> (List<ConnectorFiltering>) args[0]
+
+        );
+
+    static {
+        CONNECTOR_FILTERING_PARSER.declareObjectArray(
+            constructorArg(),
+            (p, c) -> ConnectorFiltering.fromXContent(p),
+            Connector.FILTERING_FIELD
+        );
+    }
+
+    /**
+     * Deserializes the 'filtering' field from a connector's byte representation.
```

Comment:
```suggestion
     * Deserializes the {@link ConnectorFiltering} property from a connector's byte representation.
```

Just to make it consistent with the docs for `updateConnectorFiltering` in the index service